pkgs/development/python-modules/netbox-contract/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  python-dateutil,
+  drf-yasg,
+  netbox,
+}:
+buildPythonPackage rec {
+  pname = "netbox-contract";
+  version = "2.3.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "mlebreuil";
+    repo = "netbox-proxbox";
+    tag = "v${version}";
+    hash = "";
```

Comment:
Try to add a hash and check what the CI says.

Reply:
:joy: knew I forgot something :joy: Sorry for that